spec/fixtures/proofing/aamva/responses/verification_response.xml

```diff
@@ -8,6 +8,8 @@
     </MessageAddress>
   </ControlData>
   <DriverLicenseNumberMatchIndicator>true</DriverLicenseNumberMatchIndicator>
+  <DriverLicenseIssueDateMatchIndicator>true</DriverLicenseIssueDateMatchIndicator>
```

Comment:
I'm tweaking these fixtures to contain all potential attributes, all passing. Individual tests then remove or edit their corresponding `MatchIndicator` node.